components/CheckBalanceAmount.jsx

```diff
@@ -118,8 +134,14 @@ export default () => {
                 type='submit'
                 title='Check'
                 variant='secondary'
-                mt={[2, 2, 0]}
+                mx={2}
+                px={4}
                 disabled={!filAddress}
+                bg='transparent'
+                css={`
+                  position: absolute;
+                  right: 0;
+                `}
```

Comment:
Do we need to use the `css` prop here? It's not a big deal, it just smells funny

Reply:
That was me being lazy on a Friday afternoon. Fixed.